src/components/ChangePlan/PlanCalculator.styles.js

```diff
@@ -11,3 +12,16 @@ export const PlanTabContainer = styled.section`
     }
   }
 `;
+
+export const HiddenTooltip = styled.div`
+  &.hidden {
+    display: none;
+  }
+`;
+
+export const PurchaseLink = styled(Link)`
```

Comment:
I'm not sure if we have to use `Link` here. In a future we'll redirect to a route inside the webapp, but now, we are redirecting to an external URL, so in that case, we should use `<a>` instead `<Link>`

Reply:
Ok Lu, but anyway `<Link>` renders an `<a>` element. Is it the same?

Reply:
Yes, `Link` renders an `a` but is something to use with react-router. The idea of `Link` is to avoid reload the whole page, is only to navigate between routes. If you have to go to an external URL you must to use `<a>`. In fact, in the way that you have it right now, it's not working. In the CDN it shows a link to:
https://cdn.fromdoppler.com/doppler-webapp/build3295/#/plan-selection/https://app2.fromdoppler.com/ControlPanel/AccountPreferences/UpgradeAccountStep2?IdUserTypePlan=18&fromStep1=True&IdDiscountPlan=1

Reply:
Ok , thanks Lu! ❤️